tests/Doctrine/Tests/DBAL/Functional/ConnectionTest.php

```diff
@@ -70,6 +70,40 @@ public function testTransactionNestingBehavior() : void
             $this->connection->rollBack();
             self::assertEquals(0, $this->connection->getTransactionNestingLevel());
         }
+
+        $this->connection->beginTransaction();
+        $this->connection->close();
+        $this->connection->beginTransaction();
+        self::assertEquals(1, $this->connection->getTransactionNestingLevel());
```

Comment:
Does this test need to be updated or it's better to put this in a separate test?

Reply:
The name of this test is very common  - testTransactionNestingBehavior(), and my fix affects transaction nesting behavior, so I uppdate this)

I can move this lines into a separate test and rename this to something

Reply:
Please do.